lib/browser/chrome-devtools.js

```diff
@@ -0,0 +1,121 @@
+'use strict'
+
+const { dialog, Menu } = require('electron')
+const fs = require('fs')
+const url = require('url')
+
+const ipcMainUtils = require('@electron/internal/browser/ipc-main-internal-utils')
+
+const convertToMenuTemplate = function (event, items) {
+  return items.map(function (item) {
+    const transformed = item.type === 'subMenu' ? {
+      type: 'submenu',
+      label: item.label,
+      enabled: item.enabled,
+      submenu: convertToMenuTemplate(event, item.subItems)
+    } : item.type === 'separator' ? {
+      type: 'separator'
+    } : item.type === 'checkbox' ? {
+      type: 'checkbox',
+      label: item.label,
+      enabled: item.enabled,
+      checked: item.checked
+    } : {
+      type: 'normal',
+      label: item.label,
+      enabled: item.enabled
+    }
+
+    if (item.id != null) {
```

Comment:
`!=` and `==` should be avoided, they are confusing.

Is it even possible that `item.id === null` or it's of the `undefined` type?
If you want to check if `item` has `id` property then `'id' in item` is the way.

Reply:
I don't like that either, but I just kept the original code, which was doing that

Reply:
This is sad, but ok.